### DISCUSSION TITLE
Stream can't be played

### DISCUSSION THREAD

**Discussion by z-huang** [z-huang/music#147] [Announcements]
See TeamNewPipe/NewPipe#8760.

**Comment by airon90** [comment]
It's been fixed

**Reply by z-huang** [reply]
NewPipe 0.23.1 doesn't fully fixed the problem. Waiting for NewPipe 0.23.2 to be released.

**Reply by jeffreyxuworld** [reply]
[v0.23.2](https://github.com/TeamNewPipe/NewPipe/releases/tag/v0.23.2) already released
